_posts/interactive/disorder-outbreak.html

```diff
@@ -0,0 +1,55 @@
+<section id="disorder-outbreak" class="interactive fullWidthFeatureContent">
+  <div id="disorder-outbreak__intro" class="interactive__intro">
+
+    <header class="disorder-outbreak__header">
+      <h3 class="disorder-outbreak__header-title">
+        Ipsum Lorem Dolor Health Security in the Disordered World
+      </h3>
+      <p class="disorder-outbreak__header-subtitle">
+        Lorem ipsum dolor As access to critical health services degrades, the risk of dangerous outbreaks rises, and basic preventive health measures such as childhood immunizations and reproductive and maternal health care decline or cease altogether.<br
+        /><em>Instructions are to do this thing.</em>
+      </p>
+    </header>
+
+    <ul class="disorder-outbreak__legends">
+      <div class="outbreak-legend">
+        <div class="label"><strong>Outbreak Cases:&nbsp;</strong></div>
```

Comment:
I would prefix the `label` class. That one is a little generic & could easily be overwritten.